### ISSUE TITLE
Bag order

### ISSUE THREAD

**Issue by burakoztoprak** [Jaliborc/Bagnon#492]
I don't know if it's intended or not, my bag order is wrong, when I do clean bag it starts from the last bag not the first and new items that loot goes to the first bag slot. I use the same settings on my main and it's not like this. It's really annoying if it's not intended please fix it, if it's intended please add more settings, basicly what I want is when I do clean bag I want items to stack from the first slot(top left) and I want new items that come in to go into most top left slot that is empty(so I want items to be sorted top to bottom and left to right, for example lets say first and second row from the top are full there are 3 items in the third row and they are sorted left to right, I want the new item that I loot to be placed in the 4th slot from the left). I use version 7.0.4 Here is a screenshot;
![bagnon](https://cloud.githubusercontent.com/assets/21246743/17979746/da0dac3c-6b04-11e6-8b87-89d618e6adc9.png)

**Comment by bofe112** [comment]
Here is the screenshot of how I want it to be. Note that the meat(Haunch of  Meat) from top left in the first picture and at the end of the items that are sorted in the second picture are something I bought from the vendor after doing clean up bags to show where the new items I bought were placed in both characters.
![bagnon2](https://cloud.githubusercontent.com/assets/21246743/17980210/d3c6bfe2-6b06-11e6-88b9-a23a298dc39e.png)

**Comment by psykzz** [comment]
If you reverse bag order this fixes it for me. 


![image](https://cloud.githubusercontent.com/assets/1134201/17986220/942ed8dc-6b11-11e6-9eb2-f072c3efae66.png)

**Comment by Frosttyn** [comment]
This is what mine does at times.
![bubble](https://cloud.githubusercontent.com/assets/21188689/17990989/bda8898c-6b08-11e6-85c2-c86bd1790ea0.png)
![bubble2](https://cloud.githubusercontent.com/assets/21188689/17991005/ee5310ac-6b08-11e6-8ee5-5bd594aa5714.png)

**Comment by bofe112** [comment]
When I reverse the bag order, new items go to the bottom which is irritating.
![bagnon reverse](https://cloud.githubusercontent.com/assets/21246743/18025478/ad55200a-6c33-11e6-851a-2b6762aef4ad.png)

**Comment by mrkorb** [comment]
It's not Bagnon.  This is a change by Blizzard in how WoW does bag sorting.  In the chat box type 
`/run SetSortBagsRightToLeft(true)`
and it will return to how you want it.

**Comment by Andrux51** [comment]
I think the option you're looking for may be "Reverse Sort Order" - this puts items in a given bag into the last slots first. It probably does the same thing as the script above, but as a toggle.

**Comment by mrkorb** [comment]
No, it's not the same as the script I posted above.  You'll have to create a new character to test, as this sorting behavior is new as of the 7.0.3 Legion pre-patch and only affects characters created since then.  I tried every combination of Bagnon's Reverse Sort Order and Reverse Bag Order options on my new Demon Hunter, but nothing matched the sorting behavior on all my other characters.  I had to Google the issue and found the answer in that script.  It was previously available as a toggle in the game's Interface options, but Blizzard removed it for whatever reason.

**Comment by bofe112** [comment]
By the way, /run SetSortBagsRightToLeft(true) worked for me, thanks for the info.

**Comment by Qklash** [comment]
> 
> 
> By the way, /run SetSortBagsRightToLeft(true) worked for me, thanks for the info.

For anyone still having the same issue in 2019, or classic WOW, for me the fix was

/run SetSortBagsRightToLeft(false)

**Comment by SergioT87** [comment]
Any other options that may be useful for WoW TBC Classic?
